packages/contracts/scripts/go-ffi/differential-testing.go

```diff
@@ -176,9 +176,10 @@ func DiffTestUtils() {
 
 		// Create deposit transaction
 		depositTx := makeDepositTx(from, to, value, mint, gasLimit, false, data, l1BlockHash, logIndex)
+		kromaDepTx := toKromaDepositTx(depositTx)
 
 		// RLP encode deposit transaction
-		encoded, err := types.NewTx(&depositTx).MarshalBinary()
+		encoded, err := types.NewTx(&kromaDepTx).MarshalBinary()
 		checkErr(err, "Error encoding deposit transaction")
```

Comment:
Additional function `toKromaDepositTx` is unnecessary
```suggestion

		// RLP encode deposit transaction
		encoded, err := types.NewTx(&depositTx).MarshalBinary()
		checkErr(err, "Error encoding deposit transaction")

		// [Kroma: START]
		encoded, err = derive.ToKromaDepositBytes(encoded)
		checkErr(err, "Error converting deposit transaction to KromaDepositTx")
		// [Kroma: END]
```